### DISCUSSION TITLE
having problem with the setting

### DISCUSSION THREAD

**Discussion by Mantison** [jef/streetmerchant#2722] [Help]
Deleted the stuff im not looking for, like PS ,xbox and cpu but the program still search for it and the price setting not working as well, i think dotenv is not effecting the program .
[dotenv.txt](https://github.com/jef/streetmerchant/files/6975824/dotenv.txt)

**Comment by Mr-Tech-13** [comment]
hmm, I might need more than just your dotenv. Could you upload the whole folder? (Please clear personal info before doing)
